app/models/in_person_enrollment.rb

```diff
@@ -22,6 +22,28 @@ class InPersonEnrollment < ApplicationRecord
   before_save(:on_status_updated, if: :will_save_change_to_status?)
   before_create(:set_unique_id, unless: :unique_id)
 
+  def self.pending_established(start_interval, end_interval)
+    where(status: :pending).
+      and(
+        where.not(enrollment_established_at: nil).
+        and(
+          where('enrollment_established_at <= ?', start_interval).
+          and(
+            where('enrollment_established_at >= ?', end_interval),
```

Comment:
Are these comparisons backwards?

```suggestion
          where('enrollment_established_at >= ?', start_interval).
          and(
            where('enrollment_established_at <= ?', end_interval),
```